### ISSUE TITLE
Accessibility: Focus indicator missing [All Products]

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#52134]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

There is no visible focus outline on these linked images when they receive focus.

![Image](https://github.com/user-attachments/assets/6a1ba65a-bef5-4361-9fb8-037e472c6a7f)

### Component or Page: 

Component: All Products Block, Product Collection Block, etc.     

### Name affected element:

Featured image products links

### Relevant code snippets:

```
<a href=""https://woo.equalizedigital.com/product/download-product/""><img data-testid=""product-image"" alt=""Download Product"" src=""https://woo.equalizedigital.com/wp-content/uploads/woocommerce-placeholder-300x300.png"" style=""object-fit: cover;""></a>

.wc-block-grid__products .wc-block-grid__product-image a {
    border: 0;
    box-shadow: none;
    outline: 0;
    text-decoration: none;
}

.editor-styles-wrapper .wc-block-grid__products .wc-block-grid__product .wc-block-components-product-image a, .wc-block-components-product-image a {
    border: 0;
    border-radius: inherit;
    box-shadow: none;
    outline: 0;
    text-decoration: none;
}
```

### Recommended fix:

Modify the CSS in your stylesheet to ensure the element has a 2px solid outline with a 2px outline offset on focus. Outline: 0; in the provided CSS might be the culprit for the missing focus indicator on this.

**Comment by Manussakis** [comment]
I was not able to reproduce this issue.

**Theme**: Twenty Twenty-Four

Shop page
![Image](https://github.com/user-attachments/assets/e39cd5eb-ac33-4016-a381-d8e4ae4a047c)

Recommended Products

![Image](https://github.com/user-attachments/assets/d937f04b-5ac5-434c-97c0-e666e16a12f9)


We have publish the https://github.com/woocommerce/woocommerce/pull/52353 PR that fixed a similar issue on Classic themes, but I couldn't confirm it also fixed the issue reported here due to lack of context to reproduce it.

cc: @vikrampm1

**Comment by MajoMaldonado** [comment]
This issue persists, please see the video for reference: https://drive.google.com/file/d/18_hRSZDidyixVmNjRsmDsBc2KkRElQKf/view?usp=sharing

cc: @amberhinds

**Comment by Manussakis** [comment]
@MajoMaldonado Thanks for testing it!

Could you please provide me with detailed steps to reproduce it?
I am using the Twenty Twenty-Four theme, then I created a page and tried to add the All Products block as described in the issue description, but I can't find this block. 

![Image](https://github.com/user-attachments/assets/8a6bd805-6fbd-4993-91c0-15d2c661b430)

On the other hand, I was able to add the Product Collection block, but this one doesn't have the issue:

![Image](https://github.com/user-attachments/assets/a25c2fc3-2479-4d5c-8ccd-60d408220e45)

**Comment by amberhinds** [comment]
I asked @MajoMaldonado to follow up on this.

**Comment by MajoMaldonado** [comment]
@Manussakis 
- Go to https://woo.equalizedigital.com/other-components/
- Use the TAB key to navigate trought the "All Products" section.
- Verify the Graphic links do not display a focus indicator outline.

**Comment by amberhinds** [comment]
Password for viewing [https://woo.equalizedigital.com](https://woo.equalizedigital.com/other-components/) is:
`wooa11y`

**Comment by Manussakis** [comment]
@MajoMaldonado Thanks for sharing this page: https://woo.equalizedigital.com/other-components/

I confirm I can reproduce the issue on this page, but I need to be able to recreate it locally to test my changes and to put together the steps to QA.

Could you give more details on how this page was created? Please check [this comment](https://github.com/woocommerce/woocommerce/issues/52134#issuecomment-2751177097) I left.

Thanks!

**Comment by gigitux** [comment]
Thanks for opening this issue! The All Products block (and other grid blocks) are no longer visible in the inserter (see [#45351](https://github.com/woocommerce/woocommerce/issues/45351)). I recommend focusing only on the Product Collection block.

**Comment by amberhinds** [comment]
Hi @gigitux, thanks for the context on the All Products block. Since it has been deprecated, will this issue be closed as "not going to fix"? Or will the focus issue be fixed?

@Manussakis this page was created by inserting the All Products block:

<img width="1352" alt="All products block in the editor" src="https://github.com/user-attachments/assets/64ef7938-b875-4463-8359-4fbed812344d" />

It sounds like you cannot add that block to any sites where it was not already added. If you want, I can create a user account on this testing website for you and you can pull it down locally?

**Comment by amberhinds** [comment]
Actually, @Manussakis, you might be able to get this block if you import the content into a site: [WordPress XML export file](https://drive.google.com/file/d/1--1LYFwBYSmeJKGV0C0yn8zuknFgQ81S/view?usp=sharing)

**Comment by gigitux** [comment]
> thanks for the context on the All Products block. Since it has been deprecated, will this issue be closed as "not going to fix"? Or will the focus issue be fixed?

@amberhinds Based on similar issues, it seems that for deprecated blocks, we’ve decided not to implement these improvements. The issue description also mentions the Product Collection block. Before closing this, I’d at least double-check whether the issue is present in the Product Collection block and if it is present, fix it - does it make sense?


> Actually, @Manussakis, you might be able to get this block if you import the content into a site: [WordPress XML export file](https://drive.google.com/file/d/1--1LYFwBYSmeJKGV0C0yn8zuknFgQ81S/view?usp=sharing)

You can add the All Products block adding this code via Gutenberg Code Editor:
```html
<!-- wp:woocommerce/all-products {"columns":3,"rows":3,"alignButtons":false,"contentVisibility":{"orderBy":true},"orderby":"date","layoutConfig":[["woocommerce/product-image",{"imageSizing":"cropped"}],["woocommerce/product-title"],["woocommerce/product-price"],["woocommerce/product-rating"],["woocommerce/product-button"]]} -->
<div class="wp-block-woocommerce-all-products wc-block-all-products" data-attributes="{&quot;alignButtons&quot;:false,&quot;columns&quot;:3,&quot;contentVisibility&quot;:{&quot;orderBy&quot;:true},&quot;isPreview&quot;:false,&quot;layoutConfig&quot;:[[&quot;woocommerce/product-image&quot;,{&quot;imageSizing&quot;:&quot;cropped&quot;}],[&quot;woocommerce/product-title&quot;],[&quot;woocommerce/product-price&quot;],[&quot;woocommerce/product-rating&quot;],[&quot;woocommerce/product-button&quot;]],&quot;orderby&quot;:&quot;date&quot;,&quot;rows&quot;:3}"></div>
<!-- /wp:woocommerce/all-products -->
```

**Comment by amberhinds** [comment]
It is fixed in the Product Collection block.

**Comment by gigitux** [comment]
> It is fixed in the Product Collection block.

Cool! For me, it makes sense to close this issue as not planned - what do you think?

**Comment by amberhinds** [comment]
Yes, that's fine. We'll just note it in the VPAT with the recommendation to switch to the Product Collection block for full compliance.

**Comment by gigitux** [comment]
Thanks for the feedback!

**Comment by Manussakis** [comment]
Thanks @MajoMaldonado @amberhinds @gigitux
